fleet-charts/gemma-server/values.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+image: "us-docker.pkg.dev/vertex-ai/vertex-vision-model-garden-dockers/pytorch-vllm-serve:20250312_0916_RC01"
+weight: "1b"
```

Comment:
comment which ones are available?

Reply:
Added on comment on this pattern, to use other models would require plumbing through even more values.

I was originally thinking to add a lookup table so only a single option needs to be set, but now I'm thinking it will be simpler to do that using `if / else` logic for anything an end user wouldn't set anyway.  I plan to add in a subsequent PR.